src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormArtistStep.tsx

```diff
@@ -27,9 +62,70 @@ export const MyCollectionArtworkFormArtistStep: React.FC<MyCollectionArtworkForm
 
       <Spacer y={4} />
 
-      {/* TODO: Implement screen */}
+      <Text variant={"lg"}>Select an Artist</Text>
+
+      <Spacer y={4} />
+
+      <ArtistAutoComplete
+        onError={onError}
+        onChange={value => {
+          setQuery(value)
+          console.log(value)
+        }}
+        onSelect={onSelect}
+        placeholder="Search for artists on Artsy"
```

Comment:
**note:** the text seems off to me - will check this with Product and design

Reply:
Let me know if we need to change it. [This is the Figma file](https://www.figma.com/file/AF7eVKe0eErR5gB6DDRiB2/My-Collection-Web?node-id=3215%3A48900&t=qyfcLxMfBIjz2bGh-0) with the copy.